.mergify.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# Format Ref: https://doc.mergify.io/configuration.html
+pull_request_rules:
+    - name: 'After Cirrus-CI Fails'
+      conditions:
+          - '- status-failure ~= cirrus-ci/.*'
+      actions:
+          review:
```

Comment:
Create a github code-review when the condition matches (one or more cirrus-ci tasks failed).